indico/modules/users/controllers.py

```diff
@@ -896,7 +896,16 @@ def _process_pending_users(self, results):
     def _process(self, exact, external, favorites_first, **criteria):
         matches = search_users(exact=exact, include_pending=True, external=external, **criteria)
         self.externals = {}
-        results = sorted((self._serialize_entry(entry) for entry in matches), key=itemgetter('full_name', 'email'))
+
+        def _sort_key(entry):
+            exact_match_keys = [entry[k].lower() != v.lower() for k, v in criteria.items()] if not exact else []
+            unaccent_exact_match_keys = [
+                remove_accents(entry[k].lower()) != remove_accents(v.lower())
+                for k, v in criteria.items()
+            ] if not exact else []
```

Comment:
This might deserve a comment/docstring explaining what is going on i.e. that we first sort by exact matches including the accents, then exact matches but ignoring the accents.

Actually, since the accents seem to be ignored even when you enable `Exact matches only`, we could keep the `exact_match_keys` even when `exact` is False. For example, when I search for `Tomáš` I get `Tomas` first in the results which is probably not what I want if I'm already searching using accents.

Reply:
Looks good, I'd just add a comment there ;)